core/Scheduler/Timetable.php

```diff
@@ -53,6 +53,7 @@ public function removeInactiveTasks($activeTasks)
                 unset($this->timetable[$taskName]);
             }
         }
+        $this->save();
```

Comment:
Was at first wondering if it's maybe so TasksTimetable still shows them but doesn't look like it / wouldn't make any sense. Good find :)